changelog.txt

```diff
@@ -500,7 +500,7 @@ https://hibernate.atlassian.net/projects/HHH/versions/31844
     * [HHH-14257] - An Entity A with a map collection having as index an Embeddable with a an association to the Entity A fails with a NPE
     * [HHH-14251] - Invalid SQL for @Embedded UPDATE
     * [HHH-14249] - MultiLineImport fails when script contains blank spaces or tabs at the end of the last sql statement
-
+    * [HHH-14216] - Second-level cache doesn't support @OneToOne
```

Comment:
Why is this being changed? We don't typically change existing release notes

Reply:
Because the PR has been applied to 5.5.0.Alpha1 but we forgot to close the related Jira.

Reply:
ah, thanks!